src/app/pages/OnDemandTvPage/index.test.jsx

```diff
@@ -69,6 +86,33 @@ describe('OnDemand TV Brand Page ', () => {
 
     expect(getByText('نړۍ دا وخت')).toBeInTheDocument();
   });
+
+  it('a11y - should aria-hide the title', async () => {
+    fetch.mockResponse(JSON.stringify(pashtoPageData));
+
+    const { pageData } = await getInitialData('some-ondemand-tv-path');
+    const { container } = await renderPage({
+      pageData,
+      service: 'pashto',
+    });
+
+    const hiddenHeadline = container.querySelector('strong[aria-hidden=true]');
+
+    expect(hiddenHeadline).toBeDefined();
+    expect(hiddenHeadline).toContainHTML('نړۍ دا وخت');
+  });
+
+  it('a11y - should have a "content" id on the h1', async () => {
```

Comment:
along with tabindex?

Reply:
This is just ensuring we pass the correct ID to the underlying component

That component determines whether or not tabIndex is applied - that is tested here: https://github.com/bbc/simorgh/pull/6700/files#diff-8fb1dd018541b4fd680aee0cbf38ffcfR46

Reply:
ok thank you Ryan